src/modules/module_24000.c

```diff
@@ -0,0 +1,496 @@
+/**
+ * Author......: See docs/credits.txt
+ * License.....: MIT
+ */
+
+#include <inttypes.h>
+#include "common.h"
+#include "types.h"
+#include "modules.h"
+#include "bitops.h"
+#include "convert.h"
+#include "shared.h"
+
+static const u32   ATTACK_EXEC    = ATTACK_EXEC_OUTSIDE_KERNEL;
+static const u32   DGST_POS0      = 0;
+static const u32   DGST_POS1      = 1;
+static const u32   DGST_POS2      = 2;
+static const u32   DGST_POS3      = 3;
+static const u32   DGST_SIZE      = DGST_SIZE_4_8;
+static const u32   HASH_CATEGORY  = HASH_CATEGORY_FDE;
+static const char *HASH_NAME      = "BestCrypt v4 Volume Encryption";
+static const u64   KERN_TYPE      = 24000;
+static const u32   OPTI_TYPE      = OPTI_TYPE_ZERO_BYTE;
+static const u64   OPTS_TYPE      = OPTS_TYPE_PT_GENERATE_LE
+                                  | OPTS_TYPE_ST_HEX
+                                  | OPTS_TYPE_SELF_TEST_DISABLE;
+static const u32   SALT_TYPE      = SALT_TYPE_EMBEDDED;
+static const char *ST_PASS        = "password"; // the self-test is disabled, because the original scrypt settings would create a too long startup time
+static const char *ST_HASH        = "$bcve$4$08$450886364ae094fb94b4a787e4fc2484b85f305082f63b1a$3a6ead2a1b806e51ba72efe66efde5a96ec4f4966a2155e1dbbb5eeaa55249a6430d648451ad5409d1813bedc98b55c8cd896adce3bca3d5e1d843ef9dd3a8b30ac23dcd92668113c7d9caf444acbdd5e60eaa18d36975089b24cf8459ffce00";
```

Comment:
please do **NOT** use this password.
We normally always use hashcat as the password. We should be consistent also for this type of hash.
Please update the example hash and password (ST_HASH and ST_PASS) with a new generated or extracted hash.

Thank you very much